prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -0,0 +1,323 @@
+        | sed -e "s/__EXTERNAL_PUBLIC_IP__/${GATEWAY_IP_ADDRESS}/g" \
+        | sed -e "s/__SKIP_SSL_VERIFY__/true/g" \
+        | sed -e "s/__VERSION__/0.0.1/g" \
+        | sed -e "s/__SLACK_CHANNEL_VALUE__/${KYMA_ALERTS_CHANNEL}/g" \
+        | sed -e "s#__SLACK_API_URL_VALUE__#${KYMA_ALERTS_SLACK_API_URL}#g" \
+        | sed -e "s/__.*__//g" \
+        | kubectl apply -f-
+
+    waitUntilInstallerApiAvailable
+
+    sed -e "s/__VERSION__/0.0.1/g" "${INSTALLER_CR}"  | sed -e "s/__.*__//g" | kubectl apply -f-
+    kubectl label installation/kyma-installation action=install
+    "${KYMA_SCRIPTS_DIR}"/is-installed.sh --timeout 30m
+}
+
+function installStabilityChecker() {
+    shout "Install stability-checker"
+    date
+
+	STATS_FAILING_TEST_REGEXP=${STATS_FAILING_TEST_REGEXP:-"'([0-9A-Za-z_-]+)' (?:has Failed status?|failed due to too long Running status?|failed due to too long Pending status?|failed with Unknown status?)"}
+	STATS_SUCCESSFUL_TEST_REGEXP=${STATS_SUCCESSFUL_TEST_REGEXP:-"Test of '([0-9A-Za-z_-]+)' was successful"}
+	STATS_ENABLED="true"
+
+	SC_DIR=${KYMA_SOURCES_DIR}/tools/stability-checker
+
+	kubectl create -f "${SC_DIR}/local/provisioning.yaml"
+	bash "${SC_DIR}/local/helpers/isready.sh" kyma-system app  stability-test-provisioner
+	kubectl exec stability-test-provisioner -n kyma-system --  mkdir -p /home/input
+	kubectl cp "${KYMA_SCRIPTS_DIR}/testing.sh" stability-test-provisioner:/home/input/ -n kyma-system
+	kubectl cp "${KYMA_SCRIPTS_DIR}/utils.sh" stability-test-provisioner:/home/input/ -n kyma-system
+	kubectl cp "${KYMA_SCRIPTS_DIR}/testing-common.sh" stability-test-provisioner:/home/input/ -n kyma-system
+	kubectl delete pod -n kyma-system stability-test-provisioner
+
+    # create a secret with service account used for storing logs
+    kubectl create secret generic sa-stability-fluentd-storage-writer --from-file=service-account.json=/etc/credentials/sa-stability-fluentd-storage-writer/service-account.json -n kyma-system
+
+	helm install --set clusterName="${CLUSTER_NAME}" \
+	        --set logsPersistence.enabled=true \
+	        --set slackClientWebhookUrl="${SLACK_CLIENT_WEBHOOK_URL}" \
+	        --set slackClientChannelId="${STABILITY_SLACK_CLIENT_CHANNEL_ID}" \
+	        --set slackClientToken="${SLACK_CLIENT_TOKEN}" \
+	        --set stats.enabled="${STATS_ENABLED}" \
+	        --set stats.failingTestRegexp="${STATS_FAILING_TEST_REGEXP}" \
+	        --set stats.successfulTestRegexp="${STATS_SUCCESSFUL_TEST_REGEXP}" \
+	        --set testResultWindowTime="${TEST_RESULT_WINDOW_TIME}" \
+	        "${SC_DIR}/deploy/chart/stability-checker" \
+	        --namespace=kyma-system \
+	        --name=stability-checker
+}
+
+init
+azureAuthenticating
+
+DNS_DOMAIN="$(gcloud dns managed-zones describe "${CLOUDSDK_DNS_ZONE_NAME}" --project="${CLOUDSDK_CORE_PROJECT}" --format="value(dnsName)")"
+export DOMAIN="${DNS_SUBDOMAIN}.${DNS_DOMAIN%?}"
+
+addGithubDexConnector
+cleanup
```

Comment:
I would suggest to perform cleanup before adding github dex connector  (readability)